src/components/CarouselCard/__tests__/carouselCard.spec.js

```diff
@@ -34,7 +30,7 @@ describe('<CarouselCard />', () => {
                 />
             </CarouselCard>,
         );
-        expect(component.find('a.rainbow-carousel-image').length).toBe(3);
+        expect(component.find('a[href="/"]').length).toBe(3);
```

Comment:
this selector for this case is fine, but since we wrap each element in a styled component you could do:
```
import StyledAnchor from './styled/anchor';
...
expect(StyledAnchor.length).toBe(3);
```

Reply:
nice, I think the same. but I get `a[href="/"]` as finally decision for doing a protection on href prop passed to anchor, since the CarouselImage unit test is not done yet.